specs/002-workout-tracker-sqlite-migration/quickstart.md

```
@@ -383,11 +388,13 @@ EXPLAIN QUERY PLAN SELECT * FROM workout_sessions ORDER BY date DESC;
 ```
 
 **Expected Output**:
+
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)